features/requestclose.yml

```
@@ -0,0 +1,6 @@
+name: dialog.requestClose()
+description: The `requestClose()` method of a `<dialog>` HTML element closes the dialog, firing a `close` event first, which listeners can use to prevent the dialog from closing. This differs from the `close()` method, which closes the dialog immediately.
```

This description is wrong.

```suggestion
description: The `requestClose()` method of a `<dialog>` HTML element closes the dialog, firing a `cancel` event first, which listeners can use to prevent the dialog from closing. This differs from the `close()` method, which only fires the non-cancelable `close` event.
```